R/parse_authors.R

```diff
@@ -0,0 +1,62 @@
+#' Parse the `Author` field from a DESCRIPTION file
+#' 
+#' Parse the `Author` field from a DESCRIPTION file into a `person` object
+#'
+#' @param author_string A character containing the `Author` or `Maintainer` 
+#'   field from a  `DESCRIPTION` file
+#'
+#' @return A `person` object
+#' 
+#' @importFrom utils as.person
+#' 
+#' @examples
+#' # Read from a DESCRIPTION file directly
+#' utils_description <- system.file("DESCRIPTION", package = "utils")
+#' utils_authors <- read.dcf(utils_description, "Author")
+#' 
+#' parse_authors(utils_authors)
+#' 
+#' # Read from a database of CRAN metadata
+#' cran_epidemiology_packages$Author |> 
+#'   parse_authors() |> 
+#'   unlist() |> 
+#'   unique() |> 
+#'   sort()
+#' 
+#' @export
+parse_authors <- function(author_string) {
```

Comment:
```suggestion
parse_author <- function(author_string) {
```

Maybe to be in line with the description field?